src/Extensions/FileFormFactoryTaxonomyExtension.php

```diff
@@ -38,11 +38,13 @@ public function updateFormFields(FieldList $fields, $controller, $formName, $con
             $record = $context['Record'];
 
             if (!is_a($record, Folder::class)) {
+                if ($formName === 'fileEditForm') {
```

Comment:
yes, we could do that, but it will be involving creating all the parent tabs, Root, Root.Editor. etc. If we do so, how about other sibling tabs to Root.Editor? 
As I investigated, in case of a historyForm for file, the form fields are not grouped or 'Tabbed' at all, I don't think it is worthy to mimic the fields nesting structure in a history form.

Reply:
i might be wrong saying it start with Root. it might start with a root tab "Editor" but rest of my argument is still valid. @michalkleiner

Reply:
Probably a misunderstanding on what I'm asking.
I'll commit an update.